pkg/services/ngalert/notifier/channels/wecom.go

```diff
@@ -5,40 +5,82 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"net/http"
+	"time"
 
 	"github.com/prometheus/alertmanager/template"
 	"github.com/prometheus/alertmanager/types"
+	"golang.org/x/net/context/ctxhttp"
```

Comment:
We don't use this package anywhere in the grafana codebase right now. I'd prefer to do without it if we can.